baseband/vdif/header.py

```diff
@@ -28,6 +32,41 @@
                   precision=9)
 
 
+_vdif_edv_header_classes = {}
```

Comment:
If we really worry about that, we should reinstate `VDIFHeader._vdif_edv_header_classes = VDIF_HEADER_CLASSES`. But I agree that it it sufficiently unlikely that anybody is doing this.